java/src/jmri/jmris/AbstractLightServer.java

```diff
@@ -37,7 +45,7 @@ public AbstractLightServer() {
 
     synchronized protected void addLightToList(String lightName) {
         if (!lights.containsKey(lightName)) {
-            Light li = InstanceManager.lightManagerInstance().getLight(lightName);
+            Light li = instanceManager.lightManagerInstance().getLight(lightName);
```

Comment:
This and similar changes is a fundamental change to how a basic JMRI architectural class is used. This PR should not be merged until there is a consensus around the value of this change, which _breaks_ _the_ _architecture_ (in the sense of "the unchanged code no longer meets the design structure") thousands of existing references.

Reply:
That may be true, but the design allows for a piecemeal modification to parts of the system.  It isn't an all or nothing change.